packages/coverage-istanbul/src/provider.ts

```diff
@@ -26,11 +26,12 @@ import type { CoverageMap } from 'istanbul-lib-coverage'
 import libCoverage from 'istanbul-lib-coverage'
 import libSourceMaps from 'istanbul-lib-source-maps'
 import { type Instrumenter, createInstrumenter } from 'istanbul-lib-instrument'
-// @ts-expect-error @istanbuljs/schema has no type definitions
+// @ts-expect-error missing types
 import { defaults as istanbulDefaults } from '@istanbuljs/schema'
-
 // @ts-expect-error missing types
 import _TestExclude from 'test-exclude'
+
+import { version } from '../package.json' with { type: 'json' }
```

Comment:
Verified that both providers inline this in `dist/provider.js`:

> ```js
> var version = "2.1.0-beta.4";
> ```

Reply:
Is this done by Rollup automatically or do we configure it? @sheremet-va

Reply:
this is done by a json plugin